packages/core/src/controllers/publisher.ts

```diff
@@ -104,7 +105,7 @@ export class Publisher extends IPublisher {
     if (isUndefined(request.params?.tag)) delete request.params?.tag;
     this.logger.debug(`Outgoing Relay Payload`);
     this.logger.trace({ type: "message", direction: "outgoing", request });
-    return this.relayer.provider.request(request);
+    return this.relayer.request(request);
```

Comment:
Saw your bullet on this:
> all relay requests piped through relayer

I'm assuming this is simply removing unnecessary layering/indirection for `.request` by taking `provider` out of the equation or is there more to it I'm not seeing here at first glance?